lib/src/window_manager.dart

```diff
@@ -104,6 +104,7 @@ class WindowManager {
     await _channel.invokeMethod('ensureInitialized');
   }
 
+  /// You can call this to remove title bar, also can call setTitleBarStyle(TitleBarStyle.normal) to restore it.
```

Comment:
maybe mention both `setTItleBarStyle(TitleBarStyle.normal)` and `setTitleBarStyle(TitleBarStyle.hidden)` instead of just mentioning one of them?/

Reply:
ok , I add both

Reply:
Ah, also `setAsFrameless` is not "to remove title bar". That's what `TitleBarStyle.hidden` does.
What `setAsFrameless` does is, well, removes the frame.
Basically everything except the Flutter view

Reply:
And I just noticed that `setAsFrameless` is not mentioned anywhere in the README.md

Reply:
hh, I get this method from other site

Reply:
I'm not sure, but It work well on macos 13.5.2